ambiata-p.cabal

```diff
@@ -15,8 +15,9 @@ description:           A set of standard preludes for Ambiata projects. \
 library
   build-depends:
                           base                        >= 4.6        && < 5
-                        , bifunctors                  >= 4.2        && <= 5.3
-                        , deepseq                     >= 1.3        && <= 1.5
+                        , bifunctors                  >= 4.2        && < 5.4
+                        , containers                  >= 0.5        && < 0.6
```

Comment:
Not sure what would of lower bounds been here implicitly? This is probably ok, I don't think I have seen any containers 0.4 around for a while, but I am assuming this probably works with 0.4 as well? Is it worth being a little more generous just because it is `p`?